scripts/Engineering/EnginX.py

```diff
@@ -381,17 +582,22 @@ def create_difc_zero_workspace(difc, tzero, crop_on, name):
         actual_i = correction + i
         # retrieve required workspace
         if not plot_spec_num:
```

Comment:
Can you think of a nicer more readable way to select which banks to retrieve  - I don't like L569-578. I know that it allows you to put this code in a loop (and I would want to keep the loop) but it seems like a lot of work and I think the `correction` variable is quite confusing (it adjusts the bank number to be North or South, but perhaps it's nicer just populate the `banks` list correctly in the first place - i.e. `banks = [1]` for North and `banks = [2]` for South).